1_fetch.R

```diff
@@ -85,6 +86,13 @@ p1_targets_list <- list(
   tar_target(
     p1_wqp_inventory_aoi,
     subset_inventory(p1_wqp_inventory, p1_AOI_sf, buffer_dist_m = 100)
+  ),
+
+  # Download data using output of WQP inventory  
+  tar_target(
+    p1_wqp_data_aoi,
+    fetch_wqp_data(p1_wqp_inventory_aoi, p1_char_names, 
+                   query_site_limit = 500, wqp_args = wqp_args)
```

Comment:
I was anticipating that this would be mapped so that we had a target per download group? Since we are chunking _within_ the function, if we have the last group fail, it means we have to repull ALL the data which would be frustrating. One approach would be to add a column for the download group. I did this [here](https://code.usgs.gov/wma/proxies/metals/metals-data-download/-/blob/main/1_fetch.R#L64-68) for a recent project. The function [`add_download_groups()` is here](https://code.usgs.gov/wma/proxies/metals/metals-data-download/-/blob/main/1_fetch/src/ancillary_download.R#L50)

Reply:
^ One note is that it currently isn't setup to handle more than two download groups, but I think that could fairly easily be adjusted by changing [this line](https://code.usgs.gov/wma/proxies/metals/metals-data-download/-/blob/main/1_fetch/src/ancillary_download.R#L58) (maybe using `cut()`?)